docs: some small changes (#307)
numtide/treefmt: docs/configure.md

By setting the priority fields appropriately, you can control the order in which those formatters are applied for any
files they _both happen to match on_.

## Glob patterns format

This is a variant of the Unix glob pattern. It supports all the usual
selectors such as `*` and `?`.

### Examples

-   `*.go` - match all files in the project that end with a ".go" file extension.
-   `vendor/*` - match all files under the vendor folder, recursively.

## Supported Formatters

Any formatter that follows the [spec] is supported out of the box.

Already 60+ formatters are supported.

> To find example usage, take a look at <https://github.com/numtide/treefmt-nix/tree/main/examples>.

Reviewer: Minor grammatical fix

```suggestion
To find examples, take a look at <https://github.com/numtide/treefmt-nix/tree/main/examples>.
```

> If you are a Nix users, you might also like <https://github.com/numtide/treefmt-nix> that uses Nix to pull the right formatter package as well and integrates both together in a seamless fashion.

Reviewer: ```suggestion
If you are a Nix user, you might also like <https://github.com/numtide/treefmt-nix>, which uses Nix to pull in the right formatter package and seamlessly integrates both together.
```
